command/agent/config_test.go

```diff
@@ -266,6 +266,12 @@ func TestConfig_LoadConfig(t *testing.T) {
 		t.Fatalf("bad: %#v", config)
 	}
 
+	expectedConfigFiles := []string{fh.Name()}
+	if !reflect.DeepEqual(config.Files, expectedConfigFiles) {
+		t.Errorf("Loaded configs don't match\nExpected\n%+vGot\n%+v\n",
```

Comment:
Kinda weird style: Could it instead be `t.Errorf("Loaded configs don't match: want %#v; got%#v"`
You don't need the last \n anyways.